### DISCUSSION TITLE
Would like to edit content in Umbraco from a phone

### DISCUSSION THREAD

**Discussion by ScottLittleGem** [umbraco/Umbraco-CMS#10493] [Features and ideas]
Hi,

I work for a medium sized bus operator in Greater Manchester, and have been recently using Umbraco to update our website, which has been set up through Rise Digital Media.

On a PC, it's fine to use and set up, but I'm often out of the office, and whenever I have tried to look at something using the iOS version of Chrome, the site's borders and sections overlap each other, and are only really useable when holding the phone sideways, which isn't helped when the keyboard takes up half the screen.

Is there a way that the menus etc could be either shifted to one side if using Umbraco on a mobile phone, or if the whole setup could be ported over to an iOS/ Android app, but still do the same job? I appreciate that it's not as easy as just 'making an app', but I just wanted to see if it could possibly be achievable or perceivable, as I think it could help people who are still currently working away from their office environment.

Thanks for taking the time to read this, and I look forward to any feedback.

Scott

**Comment by nul800sebastiaan** [comment]
Thanks @ScottLittleGem - it's good feedback but unfortunately we're really not set up for using Umbraco backoffice on phones at the moment. We recommend at least a tablet to make everything squeeze in a lot better.

We're working on a complete new UI at the moment and while I don't think it will focus on editing on very small devices at first, maybe we'll find a way to make it responsive enough to do so. It gets really complicated really quickly though when using editors like the grid, nested content or the block editor.

**Reply by ScottLittleGem** [reply]
Hi Sebastian,

I fully appreciate the time and effort people put into things like this, and I understand that it's not as simple as 'just changing it' to work on every platform.

If it's possible at some point in the future, I think it could be of use to some customers, but not all. As it stands, I can see where you're coming from, and it's not as if I can't access the platform, so no major concerns.

Thanks for getting back to me. 😄

**Comment by GerardoGala** [comment]
Is it possible to write a separate website app strictly for editor which is used strictly for a managing a User's blogs?

The app has a login similar to the back-office. After validation, the web app will display nothing but a blog navigation and a blog editor which takes up the whole screen.

I believe there is a big need for this. Everyone carries a phone. Not everyone carries a tablet.

Thanks!
